spyder/widgets/dependencies.py

```diff
@@ -29,32 +30,45 @@ def update_dependencies(self, dependencies):
         headers = (_("Module"), _("Package name"), _(" Required "),
                    _(" Installed "), _("Provided features"))
         self.setHeaderLabels(headers)
-        mandatory_item = QTreeWidgetItem(["Mandatory"])
+
+        # Mandatory items
+        mandatory_item = QTreeWidgetItem([_("Mandatory")])
         font = mandatory_item.font(0)
         font.setBold(True)
         mandatory_item.setFont(0, font)
-        optional_item = QTreeWidgetItem(["Optional"])
+
+        # Optional items
+        optional_item = QTreeWidgetItem([_("Optional")])
         optional_item.setFont(0, font)
-        self.addTopLevelItems([mandatory_item, optional_item])
+
+        # Spyder plugins
+        spyder_plugins = QTreeWidgetItem([_("Spyder Plugins")])
+

```

Comment:
Add `spyder_plugins.setFont(0, font)` for consistency with other headers

Reply:
Agree, thanks!

Reply:
You are correct! thanks

Reply:
Fixed!